src/api/routes/approvals.js

```diff
@@ -24,6 +28,18 @@ approvals.get('/', verifyReadScope, function (req, res, next) {
 approvals.get('/:approvalId', verifyReadScope, function (req, res, next) {
     approvals.getApproval(req, res, next, req.apiUserId, req.params.approvalId);
 });
+//get By DateRange routes
+approvals.get('/byDates/:startDate/:endDate', verifyReadScope, function (req, res, next) {
```

Comment:
@darshan-clvt , Could you add one check that validates if dates are in correct format and validates if start date shud be less that ed date.